### PR TITLE
Backport to 1.10: fix imp deprecation

### PR DESCRIPTION
**Pull request by leorochael**
This is affecting the 1.10 branch today.

### CODE REVIEW

File: src/pyramid/path.py
Comment (stevepiercy) [inline src/pyramid/path.py:20]: Fixing lint.
```suggestion
        _suffix for _suffix, _, _type in imp.get_suffixes()
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
The lint build failed due to an extra space

**Comment by stevepiercy** [comment]
@leorochael would you please sign [CONTRIBUTORS.txt](https://github.com/Pylons/pyramid/blob/1.10-branch/CONTRIBUTORS.txt), and push that commit? Thank you!

**Comment by leorochael** [comment]
> @leorochael would you please sign [CONTRIBUTORS.txt](https://github.com/Pylons/pyramid/blob/1.10-branch/CONTRIBUTORS.txt), and push that commit? Thank you!

I was hoping not to have to do that since at first I just backported @mmerickel's commit to another branch.

But I guess I added enough new code to be able to stradle old and new Python versions, so I'll add myself.

**Review by stevepiercy — APPROVED** [review]
Thank you!

**Comment by leorochael** [comment]
@stevepiercy, thank you for your review and approval.

What are the next steps for inclusion and new point release of the 1.10 branch?

**Comment by stevepiercy** [comment]
@mmerickel is release manager, and he should review this PR, too.

Assuming he approves, he mentioned last weekend that a Pyramid 2.0 final release would come within a week. I assume he will conduct a release party soon in which the 1.10-branch will be a celebrated honoree.

**Comment by mmerickel** [comment]
@leorochael LGTM - Could you please add a changelog entry for this bugfix? Thank you!

**Comment by leorochael** [comment]
@mmerickel, sorry I couldn't get to it this weekend.

Thanks for merging and adding the changelog!

**Comment by mmerickel** [comment]
No worries, I just ask out of laziness. :-) Thanks for the PR!

**Pull request merged**
